Explorer/Assets/DCL/Chat/ChatCommandsHandler.cs

```diff
@@ -0,0 +1,97 @@
+﻿using CommunicationData.URLHelpers;
+using Cysharp.Threading.Tasks;
+using ECS.SceneLifeCycle.Realm;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Threading;
+using UnityEngine;
+
+namespace DCL.Chat
+{
+    internal class ChatCommandsHandler
```

Comment:
In the future the switch will grow and it'll become a mess.
For now it's not critical but the proper way would be to generalize commands properly and execute their logic in individual classes

Reply:
Yeah, I initially was thinking of classical OOP command pattern with its `ICommand` interface and `public Execute` method, but thought it would create more allocation. So I used this "functional" adaptation of command pattern. I will try to re-work it. 
However, I am not sure we will have many chat commands, but let's see.

Reply:
I compared both improved approaches:
- OOP command pattern
- functional command pattern (but with isolation of logic in separate static class)

OOP feels better, will proceed with it (and will make caching to reduce allocations. In any case, they are low here)

Reply:
it looks extendable and encapsulated now 😊